help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,142 @@
+---
+keywords: mobile;mobile engagement destinations;LINE;LINE mobile engagement destination
+title: LINE connection
+description: The LINE destination allows you to add profiles to your Platform segment and deliver personalized experiences to connected users.
+---
+
+# [!DNL LINE] connection
+
+## Overview {#overview}
+
+[[!DNL LINE]](https://line.me/en/) is a popular communication platform that connects people, services and information and has grown from a chat app into a hub for entertainment, social, and day-to-day activities.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL LINE] Messaging API](https://developers.line.biz/en/reference/messaging-api/). You can activate profiles from your Experience Platform segments as connections within [!DNL LINE] for your business needs.
+
+[!DNL LINE] uses Bearer Tokens as the authentication mechanism to communicate with the [!DNL LINE] Messaging API. Instructions to authenticate to your [!DNL LINE] instance are further below, within [Authenticate to destination](#authenticate) section.
+
+## Use cases {#use-cases}
+
+As a marketer, you can target users in a mobile engagement destination, with segments built in [!DNL Adobe Experience Platform]. Additionally, you can deliver personalized experiences to them, based on attributes from their [!DNL Adobe Experience Platform] profiles, as soon as segments and profiles are updated in [!DNL Adobe Experience Platform].
```

Comment:
Just an FYI for the future - no need to DNL Adobe Experience Platform or Experience Platform, since these terms are added to an internal do-not-translate list.